### ISSUE TITLE
KataGo engine dies for Contributing Only

### ISSUE THREAD

**Issue by timmy354321** [sanderland/katrain#575]
Im using the latest 1.11.2 version on a Mac. The engine path for KataGo works for normal play but the engine dies when selecting that same path in the contributing mode.

**Comment by sanderland** [comment]
What does the console show? "This version of KataGo was NOT compiled with support for distributed training." ?

**Comment by timmy354321** [comment]
In the general engine settings I can set the executable path as "/usr/local/Cellar/katago/1.11.0/bin/katago" and the engine works fine in human vs AI or Ai vs Ai games. When I set that same path in the contributing settings and click start contributing the console shows ERROR: engine died unexpectedly without sending output Status 0 .

**Comment by sanderland** [comment]
I mean the console as in your terminal window.
The homebrew binary does not support contributing, which is likely the problem

**Comment by timmy354321** [comment]
Yes it’s showing that exactly. Contributing isn’t supported on a Mac I guess.
